pkg/rancher-desktop/assets/scripts/moproxy.initd

```diff
@@ -0,0 +1,75 @@
+#!/sbin/openrc-run
+
+name=moproxy
+
+description="A transparent TCP to SOCKSv5/HTTP proxy."
+
+extra_started_commands="enable disable reload"
+description_enable="Start redirecting the network traffic to the HTTP proxy."
+description_disable="Stop redirecting the network traffic to the HTTP proxy."
+description_reload="Reload the proxy list."
+
+# TCP Listen address
+: ${host:=${MOPROXY_HOST:-"::"}}
+# TCP Listen port
+: ${port:=${MOPROXY_PORT:-"2080"}}
+# List of backend proxy servers
+: ${proxy_list:=${MOPROXY_PROXYLIST:-"/etc/moproxy/proxy.ini"}}
+# Additional arguments to pass to moproxy
+: ${moproxy_args:=${MOPROXY_ARGS:-""}}
+# coma seperated list of port traffic to redirect to moproxy
+: ${ports_redirected:=${MOPROXY_REDIRECTED_PORT:-"80,443"}}
+
+command="'${MOPROXY_PEER_BINARY:-/usr/sbin/moproxy}'"
+command_args="--host ${host} --port ${port} --list ${proxy_list} ${moproxy_args}"
+command_background="yes"
+pidfile="/run/${name}.pid"
+
+MOPROXY_PEER_LOGFILE="${MOPROXY_PEER_LOGFILE:-${LOG_DIR:-/var/log}/${RC_SVCNAME}-peer.log}"
```

Comment:
I think you can drop the peer since there is only a single moproxy process and there isn't an upstream process that is running on the host e.g

```
MOPROXY_LOGFILE="${MOPROXY_LOGFILE:-${LOG_DIR:-/var/log}/${RC_SVCNAME}.log}"
```